libfreerdp/core/rdp.c

```diff
@@ -1971,7 +1986,7 @@ rdpRdp* rdp_new(rdpContext* context)
 	if (!rdp->heartbeat)
 		goto fail;
 
-	rdp->multitransport = multitransport_new();
+	rdp->multitransport = multitransport_new(rdp, INITIATE_REQUEST_PROTOCOL_UDPFECL);
 
```

Comment:
Lossy is the old version not used in first intention by the servers, I'd suggest `INITIATE_REQUEST_PROTOCOL_UDPFECR | INITIATE_REQUEST_PROTOCOL_UDPFECR`

Reply:
@hardening typo? you meant `INITIATE_REQUEST_PROTOCOL_UDPFECL | INITIATE_REQUEST_PROTOCOL_UDPFECR` right?